### PR TITLE
Update team

### PR DESCRIPTION
**Pull request by Ase020**
Screenshot 2023-12-07 140959
Screenshot 2023-12-07 141048
Updated devs headshots and linkedIn profiles

### CODE REVIEW

File: src/APP/pages/products2/data.js [hunk 1]
```diff
@@ -1,201 +1,282 @@
...
+      name: "James Otieno",
+      title: "Backend Developer",
+      headshot: james,
+      portfolio: "https://www.linkedin.com/in/james-oduor/",
+    },
+    {
+      id: 5,
+      name: "Hellen Wainaina",
+      title: "Backend Developer",
+      headshot: emmy,
```
Comment (collinskasyoki) [inline src/APP/pages/products2/data.js:232]: This is using someone else's headshot.

**Comment by collinskasyoki** [comment]
@JimmyTron, was that video on the products page approved?

File: src/APP/pages/products2/data.js (hunk 1 above)
Comment (Ase020) [reply src/APP/pages/products2/data.js:232]: Thank you for this @collinskasyoki, can you please tell her to send her headshot?

**Comment by collinskasyoki** [comment]
I believe the link issue on the learn more buttons on the landing page has been fixed in another branch @Ase020 . Right?
image

**Review by collinskasyoki — APPROVED** [review]

**Comment by Ase020** [comment]
> I believe the link issue on the learn more buttons on the landing page has been fixed in another branch @Ase020 . Right? image

They were resolved on a different PR

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Lets have all the images for the team

**Review by JimmyTron — APPROVED** [review]

**Pull request merged**